transaction.go

```diff
@@ -10,6 +11,9 @@ type Transaction struct {
 	state  *State
 	cursor *State
 
+	// An identification to distinguish transactions for logging and debugging
```

Comment:
unnecessary comment.

Reply:
fixed in https://github.com/linden/tempdb/pull/1/commits/d0180d708cdb229b7186967a1691b0df787bebec.